go.mod

```diff
@@ -154,3 +154,5 @@ require (
 	modernc.org/strutil v1.2.0 // indirect
 	modernc.org/token v1.1.0 // indirect
 )
+
+replace go.temporal.io/api => github.com/temporalio/api-go v1.43.2-0.20250117224308-f070b00264f6
```

Comment:
I think you need to take out this replace since this PR is going into main. I'll review once that is pushed and the diff is more readable!